cmake/WASINNDeps.cmake

```diff
@@ -374,6 +411,19 @@ function(wasmedge_setup_llama_target target)
       simdjson::simdjson
       mtmd
     )
+    # Add HIP-specific linking when HIP is enabled
```

Comment:
Don't write redundant comments if we can directly understand the information from the code.